### PR TITLE
[RFR] Boost Angular app performance

### LINKED ISSUE #379: Boost perf by disabling debug, but don't break tests
cf https://docs.angularjs.org/guide/production#disabling-debug-data

### PR DESCRIPTION
**Pull request by jeromemacias**
Fix #379

* [x] Boost Angular app performance by adding possibility to disable debug
* [x] Add `ng-strict-di` in demo app
* [ ] Update documentation

### CODE REVIEW

**Comment by jeromemacias** [comment]
Performance boost is slightly visible in demo app, locally on my laptop.

**Comment by jeromemacias** [comment]
Test are OK (see first commit build status).

File: src/javascripts/ng-admin.js [hunk 1]
```diff
@@ -62,7 +62,8 @@ define(function (require) {
     factory.constant('AdminDescription', new AdminDescription());
 
     var ngadmin = angular.module('ng-admin', ['main', 'crud', 'AdminDescriptionModule']);
-    ngadmin.config(function(NgAdminConfigurationProvider, AdminDescription) {
+    ngadmin.config(['NgAdminConfigurationProvider', 'AdminDescription', '$compileProvider', function(NgAdminConfigurationProvider, AdminDescription, $compileProvider) {
+        $compileProvider.debugInfoEnabled(false);
```
Comment (manuquentin) [inline src/javascripts/ng-admin.js]: Can it be configurable ?

Comment (jeromemacias) [inline src/javascripts/ng-admin.js]: Done.

**Pull request merged**